pinot-ingestion-jobs/pinot-standalone/src/main/java/org/apache/pinot/ingestion/common/PinotClusterSpec.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.ingestion.common;
+
+public class PinotClusterSpec {
```

Comment:
Could you please add java doc to explain the purpose/usage (with examples) of this class? It will be good for the users as a reference for setting up an ingestion pipeline (either here or somewhere else, with a pointer here).

Reply:
will do.

Reply:
+1.
It is not clear what host and port are. Controller/server/broker/whatever. Also, can we use a (renamed) ServerInstance here?

Reply:
Will modify this to have field `controller-uri` for now